documentation/library_guide/introduction.rst

```diff
@@ -63,6 +63,8 @@ To use Parallel API with the device execution policies, you need to install the
 Difference with Standard C++ Parallel Algorithms
 ************************************************
 
+* Function object parameters passed in to algorithms executed with device policies must provide `const` function call
```

Comment:
Even if we add that I think it shouldn't go first because it's related to the algorithms being launched with device policies. The following statement about iterators is applicable to everything making it more general.

Reply:
I agree that it would work well as the second bullet.

Reply:
Moved to second bullet.